### ISSUE TITLE
control bar has a role of group but no label, could cause confusion

### ISSUE THREAD

**Issue by gkatsev** [videojs/video.js#5134]
## Description
The control bar has a role of group but no label. This could cause confusion when a screen reader user enters the control bar since nothing would be announced beyond it being a group.
However, a potential issue with adding a label to the control bar is that if individual buttons would start announcing that label as part of their announcements, it would be a worse user experience.

**Comment by OwenEdwards** [comment]
See https://github.com/videojs/video.js/blob/master/src/js/control-bar/control-bar.js#L43-L45
